RetailCoder.VBE/Settings/HotkeySettings.cs

```diff
@@ -12,45 +12,26 @@ public interface IHotkeySettings
 
     public class HotkeySettings : IHotkeySettings, IEquatable<HotkeySettings>
     {
-        private static readonly HotkeySetting[] Defaults =
-        {
-            new HotkeySetting{Name=RubberduckHotkey.ParseAll.ToString(), IsEnabled=true, HasCtrlModifier = true, Key1="`" },
-            new HotkeySetting{Name=RubberduckHotkey.IndentProcedure.ToString(), IsEnabled=true, HasCtrlModifier = true, Key1="P" },
-            new HotkeySetting{Name=RubberduckHotkey.IndentModule.ToString(), IsEnabled=true, HasCtrlModifier = true, Key1="M" },
-            new HotkeySetting{Name=RubberduckHotkey.CodeExplorer.ToString(), IsEnabled=true, HasCtrlModifier = true, Key1="R" },
-            new HotkeySetting{Name=RubberduckHotkey.FindSymbol.ToString(), IsEnabled=true, HasCtrlModifier = true, Key1="T" },
-            new HotkeySetting{Name=RubberduckHotkey.InspectionResults.ToString(), IsEnabled=true, HasCtrlModifier = true, HasShiftModifier = true, Key1="I" },
-            new HotkeySetting{Name=RubberduckHotkey.TestExplorer.ToString(), IsEnabled=true, HasCtrlModifier = true, HasShiftModifier = true, Key1="T" },
-            new HotkeySetting{Name=RubberduckHotkey.RefactorMoveCloserToUsage.ToString(), IsEnabled=true, HasCtrlModifier = true, HasShiftModifier = true, Key1="C" },
-            new HotkeySetting{Name=RubberduckHotkey.RefactorRename.ToString(), IsEnabled=true, HasCtrlModifier = true, HasShiftModifier = true, Key1="R" },
-            new HotkeySetting{Name=RubberduckHotkey.RefactorExtractMethod.ToString(), IsEnabled=true, HasCtrlModifier = true, HasShiftModifier = true, Key1="M" },
-            new HotkeySetting{Name=RubberduckHotkey.SourceControl.ToString(), IsEnabled=true, HasCtrlModifier = true, HasShiftModifier = true, Key1="D6" },
-            new HotkeySetting{Name=RubberduckHotkey.RefactorEncapsulateField.ToString(), IsEnabled=true, HasCtrlModifier = true, HasShiftModifier = true, Key1="F" },
-            new HotkeySetting{Name=RubberduckHotkey.ExportActiveProject.ToString(), IsEnabled = true, HasCtrlModifier = true, HasShiftModifier = true, Key1="E" }
-        };
-
+        private readonly IEnumerable<HotkeySetting> _defaultSettings;
         private HashSet<HotkeySetting> _settings;
 
-        public HotkeySettings()
-        {
-            Settings = Defaults.ToArray();
-        }
-
         public HotkeySetting[] Settings
         {
-            get { return _settings.ToArray(); }
+            get => _settings?.ToArray() ?? _defaultSettings?.ToArray() ?? new HotkeySetting[0];
```

Comment:
this seems weird. the `_defaultSettings` should only be relevant when one of the following two happens:

1. The settings are reset to default settings
2. The settings are loaded from file and they're incomplete